src/test/resources/features/highlighting/inspections/unexpected_keyword_args_inspection.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Inspection for unexpected keyword arguments in section
```

Comment:
Tests fail in TC and on my local machine, see https://buildserver.labs.intellij.net/project/BioLabs_SnakeCharmPlugin?branch=features%2F%23196-warn-of-unexpected-keyword-args&mode=builds

Reply:
Fixed.